decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -93,8 +93,8 @@ def source_paths
                              default: false,
                              desc: "Do not add Puma development SSL configuration options"
 
-      # we disable the webpacker installation as we will use shakapacker
-      def webpacker_gemfile_entry
+      # we disable the shakapacker installation as we will use shakapacker
```

Comment:
This comment doesn't make sense anymore 😅 

Also we should check if this is still the case (a quick check shows that in Rails 7.1 and 7.2 there isn't any shakapacker/webpacker/webpack matches).